nemo/utils/exp_manager.py

```diff
@@ -519,13 +534,15 @@ def configure_loggers(
 
 
 class NeMoModelCheckpoint(ModelCheckpoint):
+    """ Light wrapper around Lightning's ModelCheckpoint to force a saved checkpoint on train_end
+    """
     @rank_zero_only
     def on_train_end(self, trainer, pl_module):
         filepath = os.path.join(self.dirpath, self.prefix + 'end.ckpt')
         self._save_model(filepath, trainer, pl_module)
```

Comment:
@blisc do you think it make sense to produce ".nemo" file here?

E.g. call ```pl_module.save_to(save_path=os.path.join(self.dirpath, self.prefix + 'end.nemo')```

Reply:
That would make sense. We should do that